src/cli.ts

```diff
@@ -78,6 +78,9 @@ async function calc(
     // CalculateDataRequirements doesn't make use of the calcOptions object at this point
     result = calculateDataRequirements(measureBundle);
   }
+  if (!result) {
+    throw new Error('Result is not defined');
```

Comment:
```suggestion
    throw new Error(`Could not obtain result based on outputType ${program.outputType}`);
```

Suggest rephrasing this slightly to more clearly indicate what's happening.  Technically this should only happen when none of these other `if`s get processed, which would occur if we are provided an output type that we can't understand (this really can't even happen but the error message is still valuable just incase).